packages/eslint-plugin/src/rules/use-unknown-in-catch-callback-variable.ts

```diff
@@ -0,0 +1,399 @@
+                    ];
+                  }
+
+                  return [
+                    fixer.insertTextAfter(catchVariableInner, ': unknown'),
+                  ];
+                },
+              },
+            ],
+          };
+        }
+
+        return {
+          node: catchVariableOuter,
+          suggest: [
+            {
+              messageId: 'wrongTypeAnnotationSuggestion',
+              fix: (fixer: TSESLint.RuleFixer): TSESLint.RuleFix =>
+                fixer.replaceText(catchVariableTypeAnnotation, ': unknown'),
+            },
+          ],
+        };
+      } else if (
+        catchVariableInner.type === AST_NODE_TYPES.ArrayPattern ||
+        catchVariableInner.type === AST_NODE_TYPES.ObjectPattern
+      ) {
+        return {
+          node: catchVariableOuter,
+          messageId: 'useUnknownDestructuringPattern',
+        };
+      } else if (catchVariableInner.type === AST_NODE_TYPES.RestElement) {
+        const catchVariableTypeAnnotation = catchVariableInner.typeAnnotation;
+        if (catchVariableTypeAnnotation == null) {
+          return {
+            node: catchVariableOuter,
+            suggest: [
+              {
+                messageId: 'addUnknownRestTypeAnnotationSuggestion',
+                fix: (fixer): TSESLint.RuleFix =>
+                  fixer.insertTextAfter(catchVariableInner, ': [unknown]'),
+              },
+            ],
+          };
+        }
+        return {
+          node: catchVariableOuter,
+          suggest: [
+            {
+              messageId: 'wrongRestTypeAnnotationSuggestion',
+              fix: (fixer): TSESLint.RuleFix =>
+                fixer.replaceText(catchVariableTypeAnnotation, ': [unknown]'),
+            },
+          ],
+        };
+      }
+
+      // No need to handle any other case.
+      /* istanbul ignore next */
+      return undefined;
+    }
+
+    return {
+      CallExpression(node): void {
+        // If there's no arguments, this rule doesn't apply.
+        if (node.arguments.length === 0) {
+          return;
+        }
+
+        if (!isPromiseCatchAccess(node.callee)) {
+          return;
+        }
+
+        const firstArgument = node.arguments[0];
+
+        // Deal with some special cases around spread element
+        if (firstArgument.type === AST_NODE_TYPES.SpreadElement) {
+          if (node.arguments.length === 1) {
+            if (checkWhetherShouldFlagSingleSpreadArg(firstArgument)) {
+              context.report({
+                node: firstArgument,
+                messageId: 'useUnknown',
+              });
+            }
+          } else {
+            if (checkWhetherShouldFlagMultipleSpreadArgs(node.arguments)) {
+              context.report({
+                node,
+                messageId: 'useUnknownSpreadArgs',
+              });
+            }
+          }
+          return;
+        }
+
+        // Normal cases.
```

Comment:
"Normal" here is meant to be "normal" argument rather than spread argument, as opposed to implying that this the "normal" case for the rule to check (although tbh that's true too haha). But that is certainly ambiguous from the wording!

Reworded to make it explicit that I'm talking about non-spread arguments